test/testdata/app-scenarios/docker-compose/complex/docker-compose.imported.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 items:
-- apiVersion: v1
+- apiVersion: image.openshift.io/v1
```

Comment:
If you have to do this, it means the OC client is not compatible with older servers.